packages/cluster-ui/src/store/sagas.ts

```diff
@@ -16,6 +16,14 @@ export function* sagas(cacheInvalidationPeriod?: number) {
     fork(nodesSaga, cacheInvalidationPeriod),
     fork(livenessSaga, cacheInvalidationPeriod),
     fork(sessionsSaga),
-    fork(terminateSaga),
+    fork(terminateSaga, this?.sendNotification),
   ]);
 }
+
+export type SagasContext = {
+  sendNotification: SendNotification;
+};
+
+export function initClusterUISagas(context: SagasContext) {
+  return sagas.bind(context);
```

Comment:
See note in `sessionsDetailsConnected.stories.tsx` for alternative pattern to use here. I would like avoid calling `.bind`